.github/workflows/setec_notifier.yml

```diff
@@ -0,0 +1,36 @@
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 5 * * 1,2,3,4,5'
+
+permissions:
+  contents: read  # to fetch code (actions/checkout)
```

Comment:
not sure if there's any way for this to "run" off envoyproxy/envoy but look at setec (I assume no, which means updates will be annoying but I'm starting with that anyway just to see

Reply:
ci cant run in envoy using setec without giving it permissions to do so, which apart from weakening the security here would add a lot of complexity

the easier option is what i suggested elsewhere - strip setec's main to _just_ ci and just add any necessary ci there